pkg/backup/client.go

```diff
@@ -433,7 +432,7 @@ func (bc *Client) BackupRanges(
 	ranges []rtree.Range,
 	req backuppb.BackupRequest,
 	concurrency uint,
-	updateCh glue.Progress,
+	progressCallBack func(bool),
```

Comment:
please use an enum instead of a bool here

Reply:
sure, I'll change it